testsuite/MDAnalysisTests/lib/test_util.py

```diff
@@ -221,6 +221,13 @@ def test_normal(self, vec1, vec2, value):
         assert_equal(mdamath.normal(vec1, vec2), value)
         # add more non-trivial tests
 
+    @pytest.fixture()
```

Comment:
The point of `@pytest.fixture()` is to set up a function that is run before any test function we pass it to. This is unnecessary here (and since you're not passing it anywhere this test is currently not run, which is why the issues below weren't picked up).

Reply:
@fiona-naughton , thanks for the review!

Reply:
after removing `@pytest.fixture()`, and running tests locally, `pytest` prompts `fixture a not found`, how to deal with that?
Should we use `@pytest.mark.parametrize()` instead?

Reply:
@shfrz I think the problem here is that you are passing `a` and `b` as arguments to `test_angle` when they are just variables defined within the function. So pytest is probably getting confused expecting `a` and `b` to be fixtures. Try just `def test_angle(self)` and see if that works.

Reply:
@IAlibay, I tried your suggestion and it worked fine for that part but it seems even after clipping, `mdamath.angle(a**0.5, -(a**0.5))`, returns -np.pi 

![Capturehbh](https://user-images.githubusercontent.com/39633713/77701299-f78dae80-6fdb-11ea-9472-3b5c614df138.PNG)